java/java.source.base/src/org/netbeans/api/java/source/TreeMaker.java

```diff
@@ -1224,6 +1224,10 @@ public Tree BindingPattern(CharSequence name,
                                Tree type) {
         return delegate.BindingPattern(name, type);
     }
+    
+    public Tree BindingPattern(VariableTree vt) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
Thanks for providing the comment. This method name cannot be changed because we are following the same naming convention present in the JDK 16 TreeMaker. https://github.com/openjdk/jdk/blob/master/src/jdk.compiler/share/classes/com/sun/tools/javac/tree/TreeMaker.java. 
public JCBindingPattern BindingPattern(JCVariableDecl var) {
        JCBindingPattern tree = new JCBindingPattern(var);
        tree.pos = pos;
        return tree;
    }

Reply:
Notced. 
Thanks.